app/res/values-fr/strings.xml

```diff
@@ -161,7 +161,7 @@ License.
     <string name="choose_sound">Choisissez le Son</string>
     <string name="choose_video">Choisissez la Vidéo</string>
     <string name="choose_image">Choisissez l\'Image</string>
-    <string name="selected">"Sélectioné: "</string>
+    <string name="selected" tools:ignore="PrivateResource">"Sélectioné: "</string>
```

Comment:
Curious about why the change here?

Reply:
No other changes were made here. I just resolved the warning that was shown.